### ISSUE TITLE
Security message / icons missing after php update.

### ISSUE THREAD

**Issue by wozzaa** [e107inc/e107#2616]
I've recently taken over a website and this is my first time using something like this to be honest. I believe I am actually using V1 of the software. However, this is the only place for support I have managed to find. Hoping someone may have had this issue previously and knows a fix.

The hosting company performed an update to their hardware and updated PHP. Upon moving to the new server I have had a few issues on the website. Mainly icons missing, when you right click it and ask to see the picture it displays the message. 

"Error 403 - Access Denied
** Access Denied ** - The file you selected is restricted OR your IP is on our dedicated 'Anti Spam Blacklist'"

Basically I'm just after some help, even if it is obvious and I have just been an idiot. Not had to deal with any issues previously as it's basically been flawless for years without any major updates. It's only a car forum but this is obviously causing some issues now. Checked security settings and could not see anything obviously causing an issue. The files I can see when I look via FTP they are all where they should be. I just can't seem to get them to display on the site any more.

Would love some help with this, as I said. Even if it's going to make me look an idiot and something obvious. Please go ahead.

Thanks in advance. Tom.

**Comment by CaMer0n** [comment]
@wozzaa hi Tom. What version of PHP did they upgrade you to? Anything past PHP 5.6 is not compatible with e107 v1.x.  e107 v2.x will be required. Normally these upgrades should be done prior to upgrading PHP.  PHP v1.x is no longer supported, so you'd do best to make a backup and then upgrade to v2.

**Comment by rica-carv** [comment]
@CaMer0n On the images he posted initially, they upgraded PHP to v5.6

**Comment by wozzaa** [comment]
It is 5.6 as said above. I was considering upgrading to V2 however. We have a number of plugins that were created for the site and I have been told will break if we run V2.x Kind of a future project to get it all up to date at some point.

Annoyingly when I went and changed PHP back to what it was the site was unresponsive for a number of hours until changing it back to 5.6.3

It is only an issue with things in a certain folder. Everything else is flawless. To me it seems like a security issue. But this was just me making sure. I have checked the permissions as the avatars for everyone are stored in a folder in the parent directory. Everything is the same. So I'm at a bit of a loss. So now I was going to try and basically make a new version of the folder, with a slightly changed name and see if I could get the icons to display from there. Obviously I would then need a way to update every single page with calls those icons up.
